yarn-project/p2p/src/services/peer-manager/peer_manager.ts

```diff
@@ -771,7 +771,7 @@ export class PeerManager implements PeerManagerInterface {
         //TODO: maybe hard ban these peers in the future.
         //We could allow this to happen up to N times, and then hard ban?
         //Hard ban: Disallow connection via e.g. libp2p's Gater
-        this.logger.debug(`Disconnecting peer ${peerId} who failed to respond status handshake`, {
+        this.logger.warn(`Disconnecting peer ${peerId} who failed to respond status handshake`, {
```

Comment:
Are you going to revert these changes later? We get a lot of these on testnet that's all. It spams the logs.